src/recipes/SupportedBrowser.js

```diff
@@ -0,0 +1,23 @@
+/* 
+    RECIPE: <NAME OF RECIPE>
```

Comment:
Please fill in this information and be descriptive - this will help others in the future know the intended goal of this script